### DISCUSSION TITLE
Opus in chromium like browsers

### DISCUSSION THREAD

**Discussion by vitoyucepi** [savonet/liquidsoap#3845] [General]
## Summary

I revisited issue #519 and tried to verify if it is still necessary to drop metadata in opus streams and found out that in liquidsoap since *2.0.0* it is no longer required.

@toots,
Am I right?

## Reproduction

1. `ffmpeg -f lavfi -i "sine=frequency=100:duration=10" -c:a libopus -b:a 128k -y 1.opus`
1. `main.liq`
   ```ruby
   a = single("1.opus")
   enc = %ffmpeg(
       %audio(
           codec="libopus",
           b="128k",
           samplerate=48000,
       ),
       format="ogg",
   );
   output.icecast(
       enc,
       a,
       mount="stream",
       host="icecast",
       password="hackme",
   )
   ```

## Additionally
There is also an open issue AzuraCast/AzuraCast#3597.
cc: @BusterNeece, @Vaalyn

**Comment by toots** [comment]
Very glad to hear!
